<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saisie Sigma Facies</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>366</width>
         <height>449</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="4" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Typicité</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QCheckBox" name="cfce_serie">
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="expression"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Expression :</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QCheckBox" name="typicite">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="typ_serie">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Type de série : </string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="cfce_cplx">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Type de complexe : </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="typ_cplx">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Confiance</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_nom_sf">
             <property name="text">
              <string>Nom : </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="line_edit_nom_sf"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_dim_point_2">
          <property name="title">
           <string>Composition réelle en syntaxons</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="2">
            <widget class="QComboBox" name="cb_box_compo_syntax">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QWidget" name="widget_2" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QToolButton" name="psh_btn_add_syntax">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>:/plugins/SaisieCarhab/resources/img/add_element.png</normaloff>:/plugins/SaisieCarhab/resources/img/add_element.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="psh_btn_del_syntax">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>:/plugins/SaisieCarhab/resources/img/close_job.png</normaloff>:/plugins/SaisieCarhab/resources/img/close_job.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QListWidget" name="list_syntax"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>Commentaire</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTextEdit" name="rmq"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="psh_btn_val_sf">
          <property name="text">
           <string>Valider</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>line_edit_nom_sf</tabstop>
  <tabstop>typ_cplx</tabstop>
  <tabstop>cfce_cplx</tabstop>
  <tabstop>typ_serie</tabstop>
  <tabstop>cfce_serie</tabstop>
  <tabstop>expression</tabstop>
  <tabstop>typicite</tabstop>
  <tabstop>cb_box_compo_syntax</tabstop>
  <tabstop>list_syntax</tabstop>
  <tabstop>psh_btn_add_syntax</tabstop>
  <tabstop>psh_btn_del_syntax</tabstop>
  <tabstop>rmq</tabstop>
  <tabstop>psh_btn_val_sf</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
